doc/telestrat.sgml

```diff
@@ -109,6 +150,22 @@ structures; accessing the struct fields will access the chip registers.
 
 </descrip><p>
 
+<descrip>
```

Comment:
Please put all of the I/O chip descriptions into a single `<descrip> </descrip>` list.

Reply:
Done, let me know if i did it wrong